c/bobyqa_c.f90

```diff
@@ -19,6 +19,7 @@ subroutine bobyqa_c(cobj_ptr, data_ptr, n, x, f, xl, xu, nf, rhobeg, rhoend, &
 use, non_intrinsic :: cintrf_mod, only : COBJ, CCALLBACK
```

Comment:
These were not sorted previously. In general I think cleanup things like this should be done in a separate PR (unless, for example, it was already sorted alphabetically and the PR introduced a change that would break that, then of course it makes sense to preserve the existing alphabetical order), but this one is small enough that I'll do it here (and in the other files).

Reply:
I think they were already sorted.

Reply:
bobyqa and cobyla are not, but by change the rest are https://github.com/libprima/prima/blob/main/c/bobyqa_c.f90#L21

No matter, I will update them.